reana_server/reana_admin/main.py

```diff
@@ -639,3 +640,61 @@ def set_default_quota_limit(ctx):
             resource_type=resource.type_.name,
             limit=DEFAULT_QUOTA_LIMITS.get(resource.type_.name),
         )
+
+
+@reana_admin.command("consume-queue")
+@click.option(
+    "--queue-name",
+    "-q",
+    required=True,
+    type=str,
+    help="Name of the queue that will be consumed, e.g workflow-submissions",
+)
+@click.option(
+    "--key",
+    "-k",
+    type=str,
+    help="Key that will be used to get the id value from the message body, e.g workflow_name_or_id",
+)
+@click.option(
+    "--messages-to-delete",
+    "-m",
+    multiple=True,
+    help="List of message ids to remove from the queue, e.g UUID of a workflow",
+)
+@click.option(
+    "-i",
+    "--interactive",
+    is_flag=True,
+    default=False,
+    help="Manually decide which messages to remove from the queue.",
+)
+def consume_queue(
+    queue_name: str,
+    key: Optional[str],
+    messages_to_delete: List[str],
+    interactive: bool,
+):
+    """Start consuming specified queue and remove selected messages."""
+    from reana_server.reana_admin.consumer import MessageConsumer
+
+    if key and interactive:
+        click.secho(
+            "You cannot use -i and -k optionals together. Please, select one.", fg="red"
+        )
+        return
+
+    if key and not messages_to_delete:
+        click.secho(
+            f"Please provide a list of message ids (using -m option) that need to be removed from the {queue_name} queue.",
+            fg="red",
+        )
+        return
+
+    consumer = MessageConsumer(
+        queue_name=queue_name,
+        message_key=key,
+        messages_to_delete=list(messages_to_delete),
+        is_interactive=interactive,
+    )
```

Comment:
I think we should improve the error handling when the queue specified by the user doesn't exist. Right now, the command only shows a stack trace.